src/blocks/card.css

```diff
@@ -0,0 +1,108 @@
+/* #region imac */
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
I used property box-sizing: border-box

Reply:
Not fixed. Check the link that I sent before. Display property should be above the box-sizing